internal/pproflogging/pproflogging.go

```diff
@@ -97,10 +103,41 @@ var pprofProfileRates = map[ProfileName]pprofSetRate{
 	},
 }
 
+func newProfileConfigs(wrt Writer) *ProfileConfigs {
+	q := &ProfileConfigs{
+		wrt: wrt,
+	}
```

Comment:
Shouldn't this also allocate the map q.pcm ?

Reply:
No. The nil value is used to detect if there are any profiles configured.

Reply:
Ok, never mind. I noticed that your `parseProfileConfigs()` does allocate a map and assign internally. So, this is not required.

Reply:
Sounds good. Thanks for the clarification.